src/control/taskControl.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-empty */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-undef */
+/* eslint-disable no-shadow */
```

Comment:
😔.

Reply:
I've added "linebreak-style": "off" rule since it wasn't in the file. Thanks for point it out.